test/unit/presenters/publishing_api/operational_field_presenter_test.rb

```diff
@@ -5,7 +5,7 @@ class PublishingApi::OperationalFieldPresenterTest < ActiveSupport::TestCase
     @operational_field = create(
       :operational_field,
       name: "Operational Field name",
-      description: "Operational Field description",
+      description: "Operational Field description \n\n##Some title",
```

Comment:
It seems a bit odd here to add some extra stuff and then not actually use it anywhere because we stub the govspeak renderer. Wondering if instead we follow the pattern of e.g. the [featured document presenter test](https://github.com/alphagov/whitehall/blob/230ea28ad8e4ffa542b863f68d5e29eae8bdceed/test/unit/presenters/publishing_api/featured_documents_presenter_test.rb#L36) where it just uses the govspeak renderer in the expected content

Reply:
yeah, this shouldn't have been here, didn't proof my code properly. This came from when we were seeing what it looked like when run through the `govspeak_to_html` thing. 

Excellent spot on the featured document thing, we were cribbing from the[ consultation presenter test](https://github.com/alphagov/whitehall/blob/main/test/unit/presenters/publishing_api/consultation_presenter_test.rb#L409-L421), but this is much better, have sorted, thanks :)